src/Core/Content/ImportExport/Event/ImportExportAfterProcessFinishedEvent.php

```diff
@@ -0,0 +1,40 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Content\ImportExport\Event;
+
+use Shopware\Core\Content\ImportExport\Aggregate\ImportExportLog\ImportExportLogEntity;
+use Shopware\Core\Content\ImportExport\Struct\Progress;
+use Shopware\Core\Framework\Context;
+use Shopware\Core\Framework\Log\Package;
+use Symfony\Contracts\EventDispatcher\Event;
+
+/**
+ * @internal
```

Comment:
I guess the event itself shouldn't be internal right? Because we want people to use it / subscribe to it. Maybe we could make the constructor internal to allow future changes there, like adding more data.